pkgs/applications/misc/qcad/default.nix

```
@@ -0,0 +1,92 @@
+{ boost
+, fetchFromGitHub
+, mkDerivationWith
+, muparser
+, pkgconfig
+, qmake
+, qt5
+, stdenv
+, libGLU
+}:
+
+mkDerivationWith stdenv.mkDerivation rec {
+  pname = "qcad";
+  version = "v3.24.2.1";
+
+  src = fetchFromGitHub {
+    owner = "qcad";
+    repo = "qcad";
+    rev = version;
+    sha256 = "1g295gljq051x09f4d8k586bkg3vs8z22dn3rxj6xrm6803z8zw2";
+  };
+
```

I am no qt packaging expert but I think that something like that is needed as well:
```suggestion
nativeBuildInputs = [ qt5.wrapQtAppsHook ];
```

---

Retracting that comment. It looks like it's already wrapped